File: Web/src/app/model/account/loginResponse.ts
export class LoginResponse {
  id: number;
  email: string;
  hasInvestment : boolean;
  isAdvisor : boolean;
  pendingConfirmation : boolean;
  requestedToBeAdvisor : boolean; 
  advisorName: string;
  profileUrlGuid: string;

  constructor(){
  }
}

File: Model/LoginResponse.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Auctus.Model
{
    public class LoginResponse
    {
        public int Id { get; set; }
        public string Email { get; set; }
        public bool HasInvestment { get; set; }
        public bool IsAdvisor { get; set; }
        public bool PendingConfirmation { get; set; }
        public bool RequestedToBeAdvisor { get; set; }
    }
}
